src/quo/foundations/colors.cljs

```diff
@@ -4,6 +4,9 @@
     [quo.theme :as theme]
     [react-native.platform :as platform]))
 
+(def account-colors
```

Comment:
I agree with this being moved here 👍 
maybe it would be good to add a simple test in the colors_test.cljs file that makes sure the map `customization` in this colors.cljs file has a value for each key in `account-colors`?
e.g something like -
`(every #(some? (get colors/customization %)) colors/account-colors)`
not sure it's exactly right but I think you get the jist :)